.php_cs

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
```

Comment:
Can you use the StyleCI bridge used in the other repositories (https://github.com/acmephp/core/blob/master/.php_cs) ?